lib/test/t_FisherSnedecor_std.cxx

```diff
@@ -79,7 +79,7 @@ int main(int, char *[])
     Scalar eps = 1e-5;
     Point DDF = distribution.computeDDF( point );
     fullprint << "ddf     =" << DDF << std::endl;
-    fullprint << "ddf (FD)=" << distribution.ContinuousDistribution::computeDDF(point) << std::endl;
+    fullprint << "ddf (FD)=" << distribution.DistributionImplementation::computeDDF(point) << std::endl;
```

Comment:
It doesn't like this is even needed: the test succeeds with `distribution.computeDDF`

Reply:
It was an attempt to embed a validation of the analytical formula (distribution.computeDDF()) and a finite differences approximation (distribution.DistributionImplementation::computeDDF()). I can remove it if you think it is useless.

Reply:
if you can, use sed as it shows up in about 50 tests